.github/workflows/shipjs-manual-prepare.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       github.event_name == 'issue_comment' &&
       (github.event.comment.author_association == 'member' || github.event.comment.author_association == 'owner') &&
       startsWith(github.event.comment.body, '@shipjs prepare')
-    runs-on: Ubuntu-latest
+    runs-on: ubuntu-latest
```

Comment:
![image](https://user-images.githubusercontent.com/12913535/72232009-830cbb80-35f9-11ea-99f2-38d2f2860cd1.png)

Name in old case